config.md

```diff
@@ -38,6 +38,7 @@ Each container has exactly one *root filesystem*, specified in the *root* object
 
 You can add array of mount points inside container as `mounts`.
 Each record in this array must have configuration in [runtime config](runtime-config.md#mount-configuration).
+The runtime MUST mount entries in the listed order (`mounts[0]` first, `mounts[1]` second, …).
```

Comment:
On Fri, Sep 04, 2015 at 12:02:02PM -0700, Vincent Batts wrote:
> > +The runtime MUST mount entries in the listed order (`mounts[0]` first, `mounts[1]` second, …).
> 
> Perhaps simpler to say "order matters"

That wouldn't say *how* the order matters ;).  I think it's hard to
have too much clarity, but I'm fine dropping the parenthetical if it
feels too wordy.  The sentence through “… listed order” should be
sufficient to stand alone.